src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java

```diff
@@ -1,13 +1,139 @@
 package frc.team2412.robot.subsystem;
 
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.SupplyCurrentLimitConfiguration;
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
 
 import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.I2C;
+import static frc.team2412.robot.subsystem.IntakeSubsystem.IntakeConstants.*;
 
 public class IntakeSubsystem extends SubsystemBase {
-    public IntakeSubsystem(WPI_TalonFX intakeMotor1, WPI_TalonFX intakeMotor2, DoubleSolenoid intakeSolenoid) {
 
+    // Constants
 
+    public static class IntakeConstants {
+        public static Alliance teamColor = DriverStation.getAlliance();
+
+        public static final double INTAKE_IN_SPEED = 0.5;
+        public static final double INTAKE_OUT_SPEED = -0.5;
+
+        public static final SupplyCurrentLimitConfiguration MAX_MOTOR_CURRENT = new SupplyCurrentLimitConfiguration(true, 40, 40, 500);
```

Comment:
We will start with these values for now.  Probably want to do smaller values for intake.